### DISCUSSION TITLE
ReferenceError: display is not defined at Array.process

### DISCUSSION THREAD

**Discussion by ewertonhm** [rabbitmq/rabbitmq-server#8696] [Other] [answered]
### Describe the bug

After upgrading from 3.11.16 to 3.11.17 we get the following error message in the "queues", "exchanges" and "policies" pages:

`ReferenceError: display is not defined ReferenceError: display is not defined at Array.process (eval at compile (https://URL/js/ejs-1.0.min.js:1:6654), :11:67) at EJS.render (https://URL/js/eja-1.0.min.js:1:1885) at format (https://URL/js/main.js:1176:21) at https://URL/js/main.js:463:24 at with_reqs (https://URL/js/main.js:1158:9) at https://URL/js/main.js:1154:17 at req.onreadystatechange (https://URL/js/main.js:1235:17)`

Problema also happens on 3.11.18, on 3.11.16 its working fine. 

### Reproduction steps

1. Deploy version 3.11.17/18 on kubernetes
2. Acess the management page
3. Go to /queues or /exchanges or /policies page
...

### Expected behavior

Load the page

### Additional context

Running on kubernetes as a cluster (stateful set)
Using the following additionalPlugins:
    - rabbitmq_peer_discovery_k8s
    - rabbitmq_prometheus
    - rabbitmq_management
    - rabbitmq_auth_backend_ldap
    - rabbitmq_delayed_message_exchange
    - rabbitmq_shovel
    - rabbitmq_shovel_management
    
 Using the following configs:
     additionalConfig: |
      cluster_partition_handling = pause_minority
      vm_memory_high_watermark_paging_ratio = 0.99
      vm_memory_high_watermark.relative = 0.75
      disk_free_limit.relative = 1.0
      collect_statistics_interval = 10000
      # use LDAP for authentication first
      auth_backends.1.authn = ldap
      # use internal database for authorisation
      auth_backends.1.authz = internal
      # fall back to the internal database
      auth_backends.2 = internal
      auth_ldap.servers.1=dc01...
      auth_ldap.servers.2=dc02...
      auth_ldap.servers.3=vdc01...
      auth_ldap.servers.4=vdc02...
      auth_ldap.port=636
      auth_ldap.use_ssl=true
      auth_ldap.dn_lookup_bind.user_dn = USER
      auth_ldap.dn_lookup_bind.password = PASSWORD
      auth_ldap.dn_lookup_attribute = sAMAccountName
      auth_ldap.dn_lookup_base = DC=havan,DC=corp
      log.default.level = critical

**Comment by michaelklishin** [comment]
Clear your browser cache and refresh the page.

**Reply by SaimonL** [reply, marked as answer]
I had the same issue in Docker swarm. Clearing cache and reload worked for me.
